controllers/home.js

```diff
@@ -1,15 +1,19 @@
 "use strict";
 
+const TIPS = require("../tips");
+
 
 function home(req, res) {
-  let breach = null;
+  const passwordTips = TIPS.getPasswordTips();
```

Comment:
Could get rid of the `getPasswordTips` call by doing:

`tips.js`:
```js
const TIPS = [
  {
    "image": ...
  },
  ...
]

module.exports = TIPS
```

And then in `controllers/home.js`:
```js
  res.render("monitor", {
    title: "Firefox Monitor",
    featuredBreach: featuredBreach,
    passwordTips: TIPS, 
  });
```